### ISSUE TITLE
Annealed Learning Rates 

### ISSUE THREAD

**Issue by penfoldtj** [NewcastleRSE/xray-spectroscopy-ml#41]
Would it be possible to add annealed learning rates as an option. 

I tested: https://stackoverflow.com/questions/60050586/pytorch-change-the-learning-rate-based-on-number-of-epochs

Added the following to learn.py for CNNs

scheduler = optim.lr_scheduler.StepLR(optimizer, step_size=5, gamma=0.1)
**after** 
    model.train()
    optimizer = optim.Adam(model.parameters(), lr=hyperparams["lr"])

scheduler.step()
print('Epoch-{0} lr: {1}'.format(epoch, optimizer.param_groups[0]['lr']))

**after** 
    for epoch in range(n_epoch):
        running_loss = 0

The step_size = 5 and gamma = 0.1 are the number of epochs before the learning rate is change and the multiplication factor respectively. These should be hyper parameters in the in.json.

Thanks!
